capitolweb/workers/crec_parser.py

```diff
@@ -51,6 +60,49 @@ def xpath_parse(root, paths, namespaces):
     return results
 
 
+def attribute_segments(sents, speaker_ids, record_id):
+    previous = None
+    current = None
+    segment_index = 0
+    segment_sents = []
+    segments = []
+    individual_speakers = speaker_ids.keys()
```

Comment:
so if there is no speaker in the metadata and `individual_speakers` is None, then we get an empty list for `segments`, right?

Reply:
Yes, `individual_speakers` will be an empty list and `segments` will be an empty list, too.